network/qubes-iptables

```diff
@@ -21,36 +21,29 @@
 IPTABLES=iptables
 IPTABLES_DATA_DIR=/etc/qubes
 
-if [ ! -x /sbin/$IPTABLES ]; then
+if [ ! -x "/sbin/$IPTABLES" ]; then
     echo $"${IPTABLES}: /sbin/$IPTABLES does not exist."
     exit 5
 fi
 
-start() {
+start () {
     ipt=$1
-    IPTABLES_DATA=$IPTABLES_DATA_DIR/${ipt}.rules
+    IPTABLES_DATA=$IPTABLES_DATA_DIR/qubes-${ipt}.rules
```

Comment:
```suggestion
    IPTABLES_DATA=$IPTABLES_DATA_DIR/qubes-${ipt}.nft
```
?

The service fails to start with code 6 (unfortunately no more info is logged), I guess because of this (and the other .rules below).

Then, vif-route-qubes fails too, because it can't find `qbs-raw` table.

Reply:
:woman_facepalming: oops, I don’t use sysvinit so this got no testing.

Reply:
> woman_facepalming oops, I don’t use sysvinit so this got no testing.

You don't need to. This script is called by `qubes-iptables.service`.